Automattic/woocommerce-payments
# Temporarily disable saving SEPA

Fixes #6205

#### Changes proposed in this Pull Request

Temporarily disables saving/reusing SEPA Direct Debit payment methods because we don't want to enable it for subscriptions and currently they share the same flag `is_reusable`. Check discussion p1693511033290239/1693490563.321839-slack-C048UEB37MZ





#### Testing instructions

* Enable SEPA and ensure it is not possible to save it for later or use it for new subscriptions

-------------------


**Post merge**
